frontend/components/Status.tsx

```diff
@@ -0,0 +1,65 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { ITask, Ilist } from "../src/App";
```

Comment:
These types are lacking consistency of capitalisation: `ITask` vs `Ilist` (should be `IList`).

More generally, I don't know what the `I` represents in `ITask` and `IList`, but it seems like it _could_ be redundant, so just wanted to flag that.